public/locales/en/scholarshipForm.json

```diff
@@ -0,0 +1,25 @@
+{
+  "generalDescription": "Please add the general information about the scholarship.",
+  "scholarshipName": "Scholarship Name",
+  "organization": "Organization",
+  "scholarshipLink": "Scholarship Link",
+  "deadline": "Deadline",
+  "description": "Description",
+  "tags": "Tags",
+  "awardAmount": "Award Amount",
+  "requirementsDescription": "Include information that is required for applicants to have.",
+  "noEligibilityReqs": "NO ELIGIBILITY REQUIREMENTS",
```

Comment:
This doesn't need to be in all caps btw. We can always do `t(...).toUpperCase()`.